tests/conftest.py

```diff
@@ -39,6 +40,14 @@ def mock_not_implemented(*args, **kwargs):
     monkeypatch.setattr(nmdc_geoloc_tools, "landuse_dates", mock_not_implemented)
 
 
+@pytest.fixture()
+def patch_zip_stream_service(monkeypatch):
+    def mock_zip_streamer(*args, **kwargs):
+        yield b"foo"
+
+    monkeypatch.setattr(nmdc_server.api, "stream_zip_archive", mock_zip_streamer)
```

Comment:
Is zipstreamer running in testing? If so, should it be mocked or should the test run and assert that the result is a valid zip file?

Reply:
I attempted to set this up and it turned out to be a huge pain because of https://github.com/encode/starlette/issues/1102. I don't think it's worth pursuing at the moment.